packages/react-components/react-tree/src/components/TreeItem/useTreeItemContextValues.ts

```diff
@@ -1,16 +1,13 @@
 import * as React from 'react';
-import { TreeItemContextValue } from '../../contexts';
+import type { TreeItemContextValue } from '../../contexts';
 import type { TreeItemContextValues, TreeItemState } from './TreeItem.types';
+import type { ButtonContextValue } from '@fluentui/react-button';
 
 export function useTreeItemContextValues_unstable(state: TreeItemState): TreeItemContextValues {
-  const { isActionsVisible } = state;
+  const { isActionsVisible, buttonSize } = state;
```

Comment:
_nit:_ Keep alphabetical order

```suggestion
  const { buttonSize, isActionsVisible } = state;
```